packages/message-list/theme/lumo/vaadin-message-styles.js

```diff
@@ -49,6 +49,12 @@ registerStyles(
       color: var(--lumo-secondary-text-color);
       font-size: var(--lumo-font-size-s);
     }
+
+    ::slotted([slot='avatar']) {
+      --vaadin-avatar-size: var(--lumo-size-m);
+      margin-inline-end: calc(var(--lumo-space-m) - var(--vaadin-avatar-outline-width));
+      margin-top: calc(var(--lumo-space-s) - var(--vaadin-avatar-outline-width));
```

Comment:
Is this margin-right not needed anymore?

```css
:host([dir='rtl']) {
  margin-left: calc(var(--lumo-space-m) - var(--vaadin-avatar-outline-width));
  margin-right: calc(var(--vaadin-avatar-outline-width) * -1); /* <------------ */
}
```

Reply:
`margin-right` wasn't noticable because `--vaadin-avatar-outline-width` is currently `0px`. 

If that CSS variable is supposed to be always `0px`, then we could also remove it from `margin-inline-end` and `margin-top` properties as it doesn't affect anything.

Reply:
Good point, thanks. Removed.